src/nft-card-front.ts

```diff
@@ -303,4 +307,41 @@ export class NftCardFrontTemplate extends LitElement {
       </div>
     `
   }
+
+  private getButtonTemplate() {
+
+    let btnType: string = ''
+    console.log({ ...this.state, isForSale: this.isForSale})
+
+    if (this.state.hasWeb3 /* && this.isForSale */) {
+      if (this.state.isUnlocked) {
+        if (this.state.isMatchingNetwork) {
+          if (this.state.isOwnedByAccount) {
+            // The account owns asset
+            btnType = 'manage'
+          } else {
+            // Asset is for sale and not owned by currently selected account
+            btnType = 'buy'
+          }
+        } else {
+          // Network does not match or connected to unsupported network
+          btnType = "switchNetwork" + this.state.network // TODO:
```

Comment:
I thought we were gonna have the button just link to opensea if the network is wrong

Reply:
Do you think that is better?